specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/TextAnalytics.json

```diff
@@ -47,27 +47,16 @@
         ],
         "parameters": [
           {
-            "name": "model-version",
-            "in": "query",
-            "description": "(Optional) This value indicates which model will be used for scoring. If a model-version is not specified, the API should default to the latest, non-preview version. ",
-            "required": false,
-            "type": "string"
+            "$ref": "#/parameters/ModelVersion"
           },
           {
-            "name": "showStats",
-            "in": "query",
-            "description": "(Optional) if set to true, response will contain input and document level statistics.",
-            "type": "boolean",
-            "required": false
+            "$ref": "#/parameters/ShowStats"
           },
           {
-            "in": "body",
-            "name": "input",
-            "description": "Collection of documents to analyze.",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/MultiLanguageBatchInput"
-            }
+            "$ref": "#/parameters/MultiLanguageInput"
+          },
+          {
+            "$ref": "#/parameters/StringIndexType"
```

Comment:
is string index type also being added to v3.1-preview.1? My understanding was that it was being added v3.1-preview.2 and onward

Reply:
Yes its being added to v3.1-preview.1 also. I was not in the discussions for it. @thediris might know better